gulpfile.js

```diff
@@ -14,5 +14,6 @@ var elixir = require('laravel-elixir');
 elixir(function(mix) {
     mix.copy('node_modules/@dosomething/forge/dist', 'public/assets/vendor/forge');
     mix.copy('node_modules/dosomething-modal/dist', 'public/assets/vendor/modal');
+    mix.browserify('split.js', 'public/assets/js/split.js');
```

Comment:
This should be a "main" entry point, like a `app.js` or `main.js`, that then imports your `split.js` module (so if we add any other scripts in the future, they can all easily be included in the same bundle).